metricbeat/Jenkinsfile.yml

```diff
@@ -20,8 +20,6 @@ stages:
           make check-no-changes;
     build:
         make: "RACE_DETECTOR=1 make -C metricbeat check testsuite integration-tests-environment"
-        when:
-            disabled: true     ## Disabled since Integration Testing is broken. See https://github.com/elastic/beats/issues/21961
```

Comment:
@v1v I included this change after your review, could you please take another look? Thanks!

Reply:
It works like a charm! https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-24066/detail/PR-24066/2/pipeline/602#step-700-log-1179

Reply:
Great! Thanks for looking :+1: